packages/@ngtools/webpack/src/loader.ts

```diff
@@ -262,8 +262,11 @@ function _removeModuleId(refactor: TypeScriptFileRefactor) {
 
 function _getResourceRequest(element: ts.Expression, sourceFile: ts.SourceFile) {
   if (element.kind == ts.SyntaxKind.StringLiteral) {
-    // if string, assume relative path unless it start with /
-    return `'${loaderUtils.urlToRequest((element as ts.StringLiteral).text, '')}'`;
+    const url = (element as ts.StringLiteral).text;
+    if (url[0] == '.') {
```

Comment:
Would be better to check for `./`, since it is valid to have a filename starting with `.`.